modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/logging/CorrelationLoggingTest.java

```diff
@@ -153,10 +155,14 @@ public void testRetrieveDefaultCorrelationLoggingConfigsTest() throws Exception
             "{\"name\":\"synapse\",\"enabled\":\"false\",\"properties\":[]}," +
             "{\"name\":\"method-calls\",\"enabled\":\"false\",\"properties\":[]}]}";
 
-        log.info("***d testRetrieveDefaultCorrelationLoggingConfigsTest->actualResponse: " + loggingResponse.getData());
-        log.info("***d testRetrieveDefaultCorrelationLoggingConfigsTest->expectedResponse: " + expectedResponse);
-        
-        Assert.assertEquals(loggingResponse.getData(),expectedResponse);
+        await().atMost(20, TimeUnit.SECONDS).untilAsserted(()->{
+            //Retrieve default correlation logs configs from the GET method of the configs resource in devops API
+            HttpResponse loggingResponse =
+                    HTTPSClientUtils.doGet(getStoreURLHttps() + CORRELATION_CONFIG_PATH, header);
+            log.info("***d testRetrieveDefaultCorrelationLoggingConfigsTest->actualResponse: " + loggingResponse.getData());
+            log.info("***d testRetrieveDefaultCorrelationLoggingConfigsTest->expectedResponse: " + expectedResponse);
+            Assert.assertEquals(loggingResponse.getData(),expectedResponse);
+        });
```

Comment:
Do we need this awaitality approach for this rest api call?

Reply:
Moved it to outside https://github.com/wso2/product-apim/pull/13088/commits/2c7b40a5f8ca5cab63ad8b5cb0d8bc511237cc5a